config/ocha_ai.settings.yml

```diff
@@ -38,6 +38,17 @@ plugins:
       max_tokens: 8192
       api_secret: REPLACE_WITH_SECRET
       role_arn: null
+    aws_bedrock_titan_embed_text_v2:
```

Comment:
ah

Reply:
Ok, this uses the exiting bedrock settings so should be all good, eh?

Reply:
Yes, it inherits the AWS settings from the parent plugin so there is nothing new to add.